===== Patch instructions generated by SAVER =====
[+] { Move free(*(ovl_clonepaths:osrc)) at 380 (line 210, column 4) to 377 (line 211, column 4) under true }
===== Actual patch in diff format =====
@@ -207,8 +207,8 @@ int ovl_clonepaths(struct lxc_storage *orig, struct lxc_storage *new, const char
 
 		odelta = strchr(nsrc, ':');
 		if (!odelta) {
-			free(osrc);
 			ERROR("Failed to find \":\" in \"%s\"", nsrc);
+			free(osrc);
 			return -22;
 		}
 
